apis/logging/v1/kibana_types.go

```diff
@@ -51,29 +54,33 @@ type KibanaStatus struct {
 	Deployment string `json:"deployment"`
 	// +optional
 	ReplicaSets []string `json:"replicaSets,omitempty"`
+	// The status for each of the Kibana pods for the Visualization component
 	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Kibana Status",xDescriptors="urn:alm:descriptor:com.tectonic.ui:podStatuses"
 	Pods PodStateMap `json:"pods,omitempty"`
 	// +optional
 	Conditions map[string]ClusterConditions `json:"clusterCondition,omitempty"`
 }
 
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-
-// Kibana is the Schema for the kibanas API
+// +kubebuilder:object:root=true
 // +k8s:openapi-gen=true
 // +kubebuilder:resource:path=kibanas,categories=logging,scope=Namespaced
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Management State",JSONPath=".spec.managementState",type=string
 // +kubebuilder:printcolumn:name="Replicas",JSONPath=".spec.replicas",type=integer
+// Kibana instance
+// +operator-sdk:csv:customresourcedefinitions:displayName="Kibana",resources={{Deployment,v1}}
 type Kibana struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   KibanaSpec     `json:"spec,omitempty"`
+	Spec KibanaSpec `json:"spec,omitempty"`
+
+	// TODO: Change this to KibanaStatus to enable statusDescriptor
```

Comment:
Is this a general TODO for later or something you wish to fix? If later, that won't be possible w/o making a Kibana CRD v2 and this is a big story.